src/importProvider.ts

```diff
@@ -0,0 +1,180 @@
+'use strict';
+import * as vscode from 'vscode';
+import * as Url from 'url';
+import { Uri, ViewColumn } from 'vscode';
+import * as fs from 'fs';
+import * as path from 'path';
+import { guessFileExtension } from './dataUriTextDocumentContentProvider';
+
+export async function load(sourceFilename: string) {
+    if (typeof sourceFilename == 'undefined') {
+        return;
+    }
+    if (!fs.existsSync(sourceFilename)) {
+        return;
+    }
+
+    // Compose an target filename
+    let targetFilename = sourceFilename.replace('.glb', '.gltf');
+
+    if (fs.existsSync(targetFilename)) {
+        let result = await vscode.window.showWarningMessage('.glTF already present, are you sure you want to continue?', 'Yes', 'Select Path');
```

Comment:
Likewise here, let's just always show the "Save As..." dialog.

Reply:
If there's a team that has a strong need for this kind of auto-acceptance of the default output filename, then I could imagine maybe a configuration setting to enable that for that team.  But for the majority of normal users, I would want to reliably let them see the file dialog.